prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/site-overview-page/site-overview-page.component.html

```diff
@@ -66,4 +66,37 @@
 
     </form>
   </app-page-section>
+
+  <app-page-section class="mb-5">
+    <app-page-subheader2>
+      <ng-container appPageSubheader2Title>Enter Vendor</ng-container>
+    </app-page-subheader2>
+
+    <form (ngSubmit)="saveVendor()"
+          [formGroup]="form"
+          novalidate>
+
+      <div class="row">
+        <div class="col-sm-12 col-lg-9 col-xl-5 pb-2">
+
+          <mat-form-field class="w-100">
+            <mat-select placeholder="Vendor"
+                        formControlName="vendors">
+              <mat-option *ngFor="let vend of healthAuthorityVendors"
```

Comment:
Please don't abbreviate, use `vendor` instead

Reply:
it is fixed